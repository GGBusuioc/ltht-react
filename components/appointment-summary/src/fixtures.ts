import { Encounter, EncounterStatusCode, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const AppointmentOne: Encounter = {
  id: '346ca927-67e0-e931-a2c7-115087226fg2|appointment',
  metadata: mockMetadata,
  type: [
    {
      text: 'See GP regarding sore throat',
      coding: [{ display: 'GP', code: '64706' }],
    },
  ],
  period: {
    start: {
      value: '2020-04-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  serviceProvider: {
    display: 'Beeston Village Surgery',
    typeName: 'GP',
  },
  status: EncounterStatusCode.Planned,
}

const AppointmentTwo: Encounter = {
  id: '941ca927-34e0-e911-a2c7-005087226fe4|appointment',
  metadata: mockMetadata,
  type: [
    {
      text: 'Chest x-ray for recent car accident',
      coding: [{ display: 'Hospital', code: '26706' }],
    },
  ],
  period: {
    start: {
      value: '2019-12-12T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  serviceProvider: {
    display: 'Chapel Allerton Hospital',
    typeName: 'Hospital',
  },
  status: EncounterStatusCode.Finished,
}

export { AppointmentOne, AppointmentTwo }
