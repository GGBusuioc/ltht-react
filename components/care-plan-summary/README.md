
# CarePlanSummary

<!-- STORY -->

### Import

```js
import CarePlanSummary from '@ltht-react/care-plan-summary'
```

### Usage

```jsx
<CarePlanSummary />
```
  