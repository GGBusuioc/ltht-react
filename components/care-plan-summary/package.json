{
  "name": "@ltht-react/care-plan-summary",
  "version": "0.0.2",
  "description": "ltht-react CarePlanSummary component.",
  "author": "LTHT",
  "homepage": "",
  "license": "MIT",
  "main": "lib/index.js",
  "typings": "lib/index.d.ts",
  "files": [
    "lib"
  ],
  "directories": {
    "lib": "lib"
  },
  "publishConfig": {
    "access": "public"
  },
  "scripts": {
    "clean": "rimraf lib",
    "build": "yarn build:lib",
    "watch": "tsc -w",
    "build:lib": "tsc",
    "type:check": "tsc --noEmit",
    "lint": "prettylint 'src/*.{ts,tsx}'"
  },
  "dependencies": {
    "@emotion/core": "^10.0.27",
    "@emotion/styled": "^10.0.27",
    "@ltht-react/styles": "^0.0.2",
    "@ltht-react/types": "^0.0.6",
    "@ltht-react/utils": "^0.0.6",
    "@ltht-react/widget": "^0.0.2",
    "emotion": "^10.0.27",
    "react": "^16.12.0"
  }
}
