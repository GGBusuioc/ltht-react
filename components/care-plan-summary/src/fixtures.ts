import { CarePlan, CarePlanStatusCode, PartialDateTimeKindCode, CarePlanIntentCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const CarePlanOne: CarePlan = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|careplan',
  metadata: mockMetadata,
  author: { display: 'Joe Bloggs', type: 'Test' },
  intent: CarePlanIntentCode.Plan,
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  status: CarePlanStatusCode.Active,
  title: 'Test Care Plan Summary One',
}

const CarePlanTwo: CarePlan = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|careplan',
  metadata: mockMetadata,
  author: { display: 'John Mclane', type: 'Test' },
  intent: CarePlanIntentCode.Proposal,
  period: {
    start: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2015-03-04T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  status: CarePlanStatusCode.Completed,
  title: 'Test Care Plan Summary Two',
}

export { CarePlanOne, CarePlanTwo }
