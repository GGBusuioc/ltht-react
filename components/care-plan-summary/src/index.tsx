import React from 'react'

import { CarePlan } from '@ltht-react/types'
import CarePlanSummaryItem from './molecules/care-plan-summary-item'
import { Widget, WidgetHeader, WidgetList, WidgetListItem } from '@ltht-react/widget'

const CarePlanSummary = ({ title = 'Care Plans', carePlans, clickHandler }: Props) => {
  return (
    <Widget>
      <WidgetHeader>
        <h3>{title}</h3>
      </WidgetHeader>
      <WidgetList clickable={clickHandler ? true : false}>
        {carePlans.map((carePlan, index) => (
          <WidgetListItem key={index}>
            <CarePlanSummaryItem carePlan={carePlan} clickHandler={clickHandler} />
          </WidgetListItem>
        ))}
      </WidgetList>
    </Widget>
  )
}

interface Props {
  title?: string
  carePlans: CarePlan[]
  clickHandler?(carePlan: CarePlan): void
}

export default CarePlanSummary
