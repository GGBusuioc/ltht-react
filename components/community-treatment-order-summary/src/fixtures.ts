import {
  LypftCommunityTreatmentOrder,
  CarePlanStatusCode,
  PartialDateTimeKindCode,
  CarePlanIntentCode,
  Metadata,
} from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const CommunityTreatmentOrderOne: LypftCommunityTreatmentOrder = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|communityTreatmentOrder',
  metadata: mockMetadata,
  consentToTreat: 'Consent Not Given',
  legalStatus: {
    coding: [{ display: 'Approved', code: '3309007' }],
    text: 'Approved',
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  restrictions: 'No restrictions',
}

const CommunityTreatmentOrderTwo: LypftCommunityTreatmentOrder = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|communityTreatmentOrder',
  metadata: mockMetadata,
  consentToTreat: 'Consent Given',
  legalStatus: {
    coding: [{ display: 'Pending', code: '779007' }],
    text: 'Pending',
  },
  period: {
    start: {
      value: '2020-01-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
  restrictions: 'No Restrictions',
}

export { CommunityTreatmentOrderOne, CommunityTreatmentOrderTwo }
