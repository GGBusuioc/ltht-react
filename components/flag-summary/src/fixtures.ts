import { Flag, FlagStatusCode, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const FlagOne: Flag = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard',
  metadata: mockMetadata,
  author: {
    display: 'John Smith',
    reference:
      'https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser',
    type: 'Practitioner',
    identifier: undefined,
  },
  code: {
    coding: [{ display: 'Lives in Known High Risk Area', code: '109006' }],
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Year,
    },
  },
  status: FlagStatusCode.Inactive,
}

const FlagTwo: Flag = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|personalertandhazard',
  metadata: mockMetadata,
  author: {
    display: 'John Smith',
    reference:
      'https://phoenixqa.careworks.ie/Services/api/fhir/v3/practitioner/9c1ad2c0-ec34-e911-a2c5-005056926fe4|systemuser',
    type: 'Practitioner',
    identifier: undefined,
  },
  code: {
    coding: [{ display: 'High Risk of Violence', code: '56706' }],
  },
  category: {
    coding: [{ display: 'Violence', code: '66706' }],
  },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },

  status: FlagStatusCode.Inactive,
}

export { FlagOne, FlagTwo }
