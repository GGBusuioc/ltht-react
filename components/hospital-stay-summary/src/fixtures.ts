import { HospitalStay, PartialDateTimeKindCode, Metadata } from '@ltht-react/types'

const mockMetadata: Metadata = {
  dataSources: [
    {
      display: 'Mock',
    },
  ],
}

const HospitalStayOne: HospitalStay = {
  id: '580ca927-34e0-e911-a2c7-005056926fe4|hospitalstay',
  metadata: mockMetadata,
  nhsNumber: '123456789',
  serviceProvider: { display: 'Chapel Allerton Hospital', type: 'Hospital' },
  period: {
    start: {
      value: '2013-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
    end: {
      value: '2014-02-02T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
}

const HospitalStayTwo: HospitalStay = {
  id: '690ca927-68e0-c511-b2c7-898056926ze4|hospitalstay',
  metadata: mockMetadata,
  nhsNumber: '987654321',
  serviceProvider: { display: 'Beaumont Hospital', type: 'Hospital' },
  period: {
    start: {
      value: '2020-01-01T00:00:00+00:00',
      kind: PartialDateTimeKindCode.Date,
    },
  },
}

export { HospitalStayOne, HospitalStayTwo }
