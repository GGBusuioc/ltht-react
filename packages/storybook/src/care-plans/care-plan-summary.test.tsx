import React from 'react'
import { mount } from 'enzyme'
import ReactDOM from 'react-dom'

import CarePlanSummary from '@ltht-react/care-plan-summary'
import { CarePlanOne, CarePlanTwo } from './care-plan-summary.fixtures'

describe('CarePlanSummary', () => {
  const carePlans = [CarePlanOne, CarePlanTwo]
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<CarePlanSummary title="Care Plans" carePlans={carePlans} />, div)
  })
  it('matches snapshot', () => {
    expect(mount(<CarePlanSummary title="Care Plans" carePlans={carePlans} />)).toMatchSnapshot('wrapper mount')
  })
})
