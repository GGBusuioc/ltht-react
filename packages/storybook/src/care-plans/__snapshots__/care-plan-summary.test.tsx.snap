// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarePlanSummary matches snapshot: wrapper mount 1`] = `
.emotion-16 {
  color: #231f20;
  font-family: 'SF Pro Text','SF Pro Icons','Helvetica Neue',Helvetica,Arial,sans-serif;
  font-size: 0.8rem;
  background: #FFFFFF;
  color: #231f20;
  margin-bottom: 0.5rem;
  border-radius: 4px;
  box-shadow: 0px 2px 1px -1px rgba(102,102,102,0.1), 0px 1px 1px 0px rgba(102,102,102,0.15), 0px 1px 3px 0px rgba(102,102,102,.6);
  margin: 0 5px 10px 0;
  -webkit-font-smoothing: antialiased;
}

.emotion-16 *,
.emotion-16 ::before,
.emotion-16 ::after {
  box-sizing: border-box;
  padding: 0;
  margin: 0;
}

.emotion-16 * {
  margin: 0;
  padding: 0;
}

.emotion-16 h3 {
  font-size: 1rem;
}

@media only screen and (min-width:768px) {
  .emotion-16 {
    font-size: 0.9rem;
  }

  .emotion-16 h3 {
    font-size: 1.1rem;
  }
}

@media only screen and (min-width:1024px) {
  .emotion-16 {
    font-size: 1rem;
  }

  .emotion-16 h3 {
    font-size: 1.2rem;
  }
}

@media only screen and (min-width:1200px) {
  .emotion-16 {
    font-size: 1.1rem;
  }

  .emotion-16 h3 {
    font-size: 1.3rem;
  }
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 0.5rem;
}

.emotion-2:hover {
  cursor: pointer;
}

.emotion-0 {
  color: #768692;
}

.emotion-15 {
  list-style: none;
  padding-left: 0.5rem;
}

.emotion-15 li {
  border-top: 1px solid #b0b0b0;
  padding: 0.5rem 0.5rem 0.5rem 0;
}

.emotion-8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.emotion-5 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.emotion-3 {
  color: #231f20;
  text-align: left;
}

.emotion-4 {
  color: #768692;
  text-align: left;
}

.emotion-6 {
  color: #231f20;
  text-align: right;
}

.emotion-7 {
  color: #768692;
  text-align: right;
}

<CarePlanSummary
  carePlans={
    Array [
      Object {
        "author": Array [
          Object {
            "display": "Joe Bloggs",
            "typeName": "Test",
          },
        ],
        "careTeam": Array [
          Object {
            "display": "LYPFT Care Team 1",
            "typeName": "Team",
          },
        ],
        "id": "580ca927-34e0-e911-a2c7-005056926fe4|careplan",
        "intent": "PLAN",
        "metadata": Object {
          "dataSources": Array [
            Object {
              "display": "Mock",
            },
          ],
        },
        "period": Object {
          "end": Object {
            "kind": "DATE",
            "value": "2014-02-02T00:00:00+00:00",
          },
          "start": Object {
            "kind": "DATE",
            "value": "2013-01-01T00:00:00+00:00",
          },
        },
        "status": "ACTIVE",
        "title": "Mental Health Crisis Plan",
      },
      Object {
        "author": Array [
          Object {
            "display": "John Mclane",
            "typeName": "Test",
          },
        ],
        "careTeam": Array [
          Object {
            "display": "LYPFT Care Team 2",
            "typeName": "Team",
          },
        ],
        "id": "690ca927-68e0-c511-b2c7-898056926ze4|careplan",
        "intent": "PROPOSAL",
        "metadata": Object {
          "dataSources": Array [
            Object {
              "display": "Mock",
            },
          ],
        },
        "period": Object {
          "end": Object {
            "kind": "DATE",
            "value": "2015-03-04T00:00:00+00:00",
          },
          "start": Object {
            "kind": "DATE",
            "value": "2014-02-02T00:00:00+00:00",
          },
        },
        "status": "COMPLETED",
        "title": "Test Care Plan",
      },
    ]
  }
  title="Care Plans"
>
  <Widget
    noData={false}
  >
    <WidgetProvider
      collapsible={true}
      noData={false}
    >
      <div
        className="emotion-16"
      >
        <WidgetHeader>
          <div
            className="emotion-2"
            onClick={[Function]}
            title="Click to collapse"
          >
            <div>
              <h3>
                Care Plans
              </h3>
            </div>
            <div>
              <ChevronCircleIcon
                direction="down"
                size="medium"
              >
                <FontAwesomeIcon
                  border={false}
                  className="emotion-0"
                  fixedWidth={false}
                  flip={null}
                  icon={
                    Object {
                      "icon": Array [
                        512,
                        512,
                        Array [],
                        "f13a",
                        "M504 256c0 137-111 248-248 248S8 393 8 256 119 8 256 8s248 111 248 248zM273 369.9l135.5-135.5c9.4-9.4 9.4-24.6 0-33.9l-17-17c-9.4-9.4-24.6-9.4-33.9 0L256 285.1 154.4 183.5c-9.4-9.4-24.6-9.4-33.9 0l-17 17c-9.4 9.4-9.4 24.6 0 33.9L239 369.9c9.4 9.4 24.6 9.4 34 0z",
                      ],
                      "iconName": "chevron-circle-down",
                      "prefix": "fas",
                    }
                  }
                  inverse={false}
                  listItem={false}
                  mask={null}
                  pull={null}
                  pulse={false}
                  rotation={null}
                  size="sm"
                  spin={false}
                  swapOpacity={false}
                  symbol={false}
                  title=""
                  transform={null}
                >
                  <svg
                    aria-hidden="true"
                    className="svg-inline--fa fa-chevron-circle-down fa-w-16 fa-sm emotion-0"
                    data-icon="chevron-circle-down"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style={Object {}}
                    viewBox="0 0 512 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M504 256c0 137-111 248-248 248S8 393 8 256 119 8 256 8s248 111 248 248zM273 369.9l135.5-135.5c9.4-9.4 9.4-24.6 0-33.9l-17-17c-9.4-9.4-24.6-9.4-33.9 0L256 285.1 154.4 183.5c-9.4-9.4-24.6-9.4-33.9 0l-17 17c-9.4 9.4-9.4 24.6 0 33.9L239 369.9c9.4 9.4 24.6 9.4 34 0z"
                      fill="currentColor"
                      style={Object {}}
                    />
                  </svg>
                </FontAwesomeIcon>
              </ChevronCircleIcon>
            </div>
          </div>
        </WidgetHeader>
        <WidgetList
          clickable={false}
        >
          <ul
            className="emotion-15"
          >
            <WidgetListItem
              key="580ca927-34e0-e911-a2c7-005056926fe4|careplan"
            >
              <li>
                <CarePlanSummaryItem
                  carePlan={
                    Object {
                      "author": Array [
                        Object {
                          "display": "Joe Bloggs",
                          "typeName": "Test",
                        },
                      ],
                      "careTeam": Array [
                        Object {
                          "display": "LYPFT Care Team 1",
                          "typeName": "Team",
                        },
                      ],
                      "id": "580ca927-34e0-e911-a2c7-005056926fe4|careplan",
                      "intent": "PLAN",
                      "metadata": Object {
                        "dataSources": Array [
                          Object {
                            "display": "Mock",
                          },
                        ],
                      },
                      "period": Object {
                        "end": Object {
                          "kind": "DATE",
                          "value": "2014-02-02T00:00:00+00:00",
                        },
                        "start": Object {
                          "kind": "DATE",
                          "value": "2013-01-01T00:00:00+00:00",
                        },
                      },
                      "status": "ACTIVE",
                      "title": "Mental Health Crisis Plan",
                    }
                  }
                >
                  <div
                    className="emotion-8"
                    role="link"
                  >
                    <div
                      className="emotion-5"
                    >
                      <CarePlanTitle
                        carePlan={
                          Object {
                            "author": Array [
                              Object {
                                "display": "Joe Bloggs",
                                "typeName": "Test",
                              },
                            ],
                            "careTeam": Array [
                              Object {
                                "display": "LYPFT Care Team 1",
                                "typeName": "Team",
                              },
                            ],
                            "id": "580ca927-34e0-e911-a2c7-005056926fe4|careplan",
                            "intent": "PLAN",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                            },
                            "period": Object {
                              "end": Object {
                                "kind": "DATE",
                                "value": "2014-02-02T00:00:00+00:00",
                              },
                              "start": Object {
                                "kind": "DATE",
                                "value": "2013-01-01T00:00:00+00:00",
                              },
                            },
                            "status": "ACTIVE",
                            "title": "Mental Health Crisis Plan",
                          }
                        }
                      >
                        <div
                          className="emotion-3"
                        >
                          Mental Health Crisis Plan
                        </div>
                      </CarePlanTitle>
                      <CarePlanDescription
                        carePlan={
                          Object {
                            "author": Array [
                              Object {
                                "display": "Joe Bloggs",
                                "typeName": "Test",
                              },
                            ],
                            "careTeam": Array [
                              Object {
                                "display": "LYPFT Care Team 1",
                                "typeName": "Team",
                              },
                            ],
                            "id": "580ca927-34e0-e911-a2c7-005056926fe4|careplan",
                            "intent": "PLAN",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                            },
                            "period": Object {
                              "end": Object {
                                "kind": "DATE",
                                "value": "2014-02-02T00:00:00+00:00",
                              },
                              "start": Object {
                                "kind": "DATE",
                                "value": "2013-01-01T00:00:00+00:00",
                              },
                            },
                            "status": "ACTIVE",
                            "title": "Mental Health Crisis Plan",
                          }
                        }
                      >
                        <div
                          className="emotion-4"
                        >
                          Joe Bloggs - LYPFT Care Team 1
                        </div>
                      </CarePlanDescription>
                    </div>
                    <div>
                      <CarePlanDate
                        carePlan={
                          Object {
                            "author": Array [
                              Object {
                                "display": "Joe Bloggs",
                                "typeName": "Test",
                              },
                            ],
                            "careTeam": Array [
                              Object {
                                "display": "LYPFT Care Team 1",
                                "typeName": "Team",
                              },
                            ],
                            "id": "580ca927-34e0-e911-a2c7-005056926fe4|careplan",
                            "intent": "PLAN",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                            },
                            "period": Object {
                              "end": Object {
                                "kind": "DATE",
                                "value": "2014-02-02T00:00:00+00:00",
                              },
                              "start": Object {
                                "kind": "DATE",
                                "value": "2013-01-01T00:00:00+00:00",
                              },
                            },
                            "status": "ACTIVE",
                            "title": "Mental Health Crisis Plan",
                          }
                        }
                      >
                        <div
                          className="emotion-6"
                        >
                          01-Jan-2013 to 02-Feb-2014
                        </div>
                      </CarePlanDate>
                      <CarePlanStatus
                        carePlan={
                          Object {
                            "author": Array [
                              Object {
                                "display": "Joe Bloggs",
                                "typeName": "Test",
                              },
                            ],
                            "careTeam": Array [
                              Object {
                                "display": "LYPFT Care Team 1",
                                "typeName": "Team",
                              },
                            ],
                            "id": "580ca927-34e0-e911-a2c7-005056926fe4|careplan",
                            "intent": "PLAN",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                            },
                            "period": Object {
                              "end": Object {
                                "kind": "DATE",
                                "value": "2014-02-02T00:00:00+00:00",
                              },
                              "start": Object {
                                "kind": "DATE",
                                "value": "2013-01-01T00:00:00+00:00",
                              },
                            },
                            "status": "ACTIVE",
                            "title": "Mental Health Crisis Plan",
                          }
                        }
                      >
                        <div
                          className="emotion-7"
                        >
                          Plan - Active
                        </div>
                      </CarePlanStatus>
                    </div>
                  </div>
                </CarePlanSummaryItem>
              </li>
            </WidgetListItem>
            <WidgetListItem
              key="690ca927-68e0-c511-b2c7-898056926ze4|careplan"
            >
              <li>
                <CarePlanSummaryItem
                  carePlan={
                    Object {
                      "author": Array [
                        Object {
                          "display": "John Mclane",
                          "typeName": "Test",
                        },
                      ],
                      "careTeam": Array [
                        Object {
                          "display": "LYPFT Care Team 2",
                          "typeName": "Team",
                        },
                      ],
                      "id": "690ca927-68e0-c511-b2c7-898056926ze4|careplan",
                      "intent": "PROPOSAL",
                      "metadata": Object {
                        "dataSources": Array [
                          Object {
                            "display": "Mock",
                          },
                        ],
                      },
                      "period": Object {
                        "end": Object {
                          "kind": "DATE",
                          "value": "2015-03-04T00:00:00+00:00",
                        },
                        "start": Object {
                          "kind": "DATE",
                          "value": "2014-02-02T00:00:00+00:00",
                        },
                      },
                      "status": "COMPLETED",
                      "title": "Test Care Plan",
                    }
                  }
                >
                  <div
                    className="emotion-8"
                    role="link"
                  >
                    <div
                      className="emotion-5"
                    >
                      <CarePlanTitle
                        carePlan={
                          Object {
                            "author": Array [
                              Object {
                                "display": "John Mclane",
                                "typeName": "Test",
                              },
                            ],
                            "careTeam": Array [
                              Object {
                                "display": "LYPFT Care Team 2",
                                "typeName": "Team",
                              },
                            ],
                            "id": "690ca927-68e0-c511-b2c7-898056926ze4|careplan",
                            "intent": "PROPOSAL",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                            },
                            "period": Object {
                              "end": Object {
                                "kind": "DATE",
                                "value": "2015-03-04T00:00:00+00:00",
                              },
                              "start": Object {
                                "kind": "DATE",
                                "value": "2014-02-02T00:00:00+00:00",
                              },
                            },
                            "status": "COMPLETED",
                            "title": "Test Care Plan",
                          }
                        }
                      >
                        <div
                          className="emotion-3"
                        >
                          Test Care Plan
                        </div>
                      </CarePlanTitle>
                      <CarePlanDescription
                        carePlan={
                          Object {
                            "author": Array [
                              Object {
                                "display": "John Mclane",
                                "typeName": "Test",
                              },
                            ],
                            "careTeam": Array [
                              Object {
                                "display": "LYPFT Care Team 2",
                                "typeName": "Team",
                              },
                            ],
                            "id": "690ca927-68e0-c511-b2c7-898056926ze4|careplan",
                            "intent": "PROPOSAL",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                            },
                            "period": Object {
                              "end": Object {
                                "kind": "DATE",
                                "value": "2015-03-04T00:00:00+00:00",
                              },
                              "start": Object {
                                "kind": "DATE",
                                "value": "2014-02-02T00:00:00+00:00",
                              },
                            },
                            "status": "COMPLETED",
                            "title": "Test Care Plan",
                          }
                        }
                      >
                        <div
                          className="emotion-4"
                        >
                          John Mclane - LYPFT Care Team 2
                        </div>
                      </CarePlanDescription>
                    </div>
                    <div>
                      <CarePlanDate
                        carePlan={
                          Object {
                            "author": Array [
                              Object {
                                "display": "John Mclane",
                                "typeName": "Test",
                              },
                            ],
                            "careTeam": Array [
                              Object {
                                "display": "LYPFT Care Team 2",
                                "typeName": "Team",
                              },
                            ],
                            "id": "690ca927-68e0-c511-b2c7-898056926ze4|careplan",
                            "intent": "PROPOSAL",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                            },
                            "period": Object {
                              "end": Object {
                                "kind": "DATE",
                                "value": "2015-03-04T00:00:00+00:00",
                              },
                              "start": Object {
                                "kind": "DATE",
                                "value": "2014-02-02T00:00:00+00:00",
                              },
                            },
                            "status": "COMPLETED",
                            "title": "Test Care Plan",
                          }
                        }
                      >
                        <div
                          className="emotion-6"
                        >
                          02-Feb-2014 to 04-Mar-2015
                        </div>
                      </CarePlanDate>
                      <CarePlanStatus
                        carePlan={
                          Object {
                            "author": Array [
                              Object {
                                "display": "John Mclane",
                                "typeName": "Test",
                              },
                            ],
                            "careTeam": Array [
                              Object {
                                "display": "LYPFT Care Team 2",
                                "typeName": "Team",
                              },
                            ],
                            "id": "690ca927-68e0-c511-b2c7-898056926ze4|careplan",
                            "intent": "PROPOSAL",
                            "metadata": Object {
                              "dataSources": Array [
                                Object {
                                  "display": "Mock",
                                },
                              ],
                            },
                            "period": Object {
                              "end": Object {
                                "kind": "DATE",
                                "value": "2015-03-04T00:00:00+00:00",
                              },
                              "start": Object {
                                "kind": "DATE",
                                "value": "2014-02-02T00:00:00+00:00",
                              },
                            },
                            "status": "COMPLETED",
                            "title": "Test Care Plan",
                          }
                        }
                      >
                        <div
                          className="emotion-7"
                        >
                          Proposal - Completed
                        </div>
                      </CarePlanStatus>
                    </div>
                  </div>
                </CarePlanSummaryItem>
              </li>
            </WidgetListItem>
          </ul>
        </WidgetList>
      </div>
    </WidgetProvider>
  </Widget>
</CarePlanSummary>
`;
